# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/rarian/hotfix-autogen.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- rarian-0.8.6/autogen.sh.vanilla	2026-09-28 12:50:22.203548704 +0300
+++ rarian-0.8.6/autogen.sh	2026-09-28 12:50:35.286846087 +0300
@@ -414,8 +414,8 @@
 
 if test x$NOCONFIGURE = x; then
     printbold Running $srcdir/configure $conf_flags "$@" ...
-    $srcdir/configure $conf_flags "$@" \
-	&& echo Now type \`make\' to compile $PKG_NAME || exit 1
+    $srcdir/configure $conf_flags "$@" && \
+	echo Now type \`make\' to compile $PKG_NAME || exit 1
 else
     echo Skipping configure process.
 fi
